core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -320,6 +320,11 @@ trait AttributeComponent {
       }
     }
 
+    val caseSensitiveCollate = SimpleExpression.unary[Option[String], Option[String]] { (value, qb) => 
+      qb.expr(value)
+      qb.sqlBuilder += " collate utf8_bin"
```

Comment:
How did this perform while testing? I'm curious about the before/after `explain` plan of this query with this collate added. 

In case it helps, you can enable the raw SQL output of these Slick queries in the logs when running locally. Just uncomment the `slick.jdbc.JdbcBackend.statement` logger in `logback.xml` (around line 100)